RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitPhase2WeightsAlgoGPU.cu

```diff
@@ -0,0 +1,35 @@
+#include "DataFormats/EcalDigi/interface/EcalDataFrame_Ph2.h"
+#include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
+
+#include "EcalUncalibRecHitPhase2WeightsKernels.h"
+#include "EcalUncalibRecHitPhase2WeightsAlgoGPU.h"
+
+namespace ecal {
+  namespace weights {
+
+    void entryPoint(ecal::DigisCollection<calo::common::DevStoragePolicy> const& digis,
```

Comment:
Could you rename this to be more descriptive, like `phase2Weights()`, `runPhase2Weights()`, _etc._ ?

(I know the other ECAL and HCAL code use plain `entryPoint()`, I realised it while trying to debug some ECAL-related issues, and realised how much more difficult that makes it to find the right place to looks at ...)